lighthouse-core/config/default-config.js

```diff
@@ -105,6 +105,14 @@ const UIStrings = {
   'not automatically checked by Lighthouse. They do not affect your score but it\'s important that you verify them manually.',
   /** Title of the Best Practices category of audits. This is displayed at the top of a list of audits focused on topics related to following web development best practices and accepted guidelines. Also used as a label of a score gauge; try to limit to 20 characters. */
   bestPracticesCategoryTitle: 'Best Practices',
+  /** Title of the Trust & Safety group of the Best Practices category. Within this section are the audits related to trust and safety. */
+  bestPracticesSafetyGroupTitle: 'Trust & Safety',
+  /** Title of the User Experience group of the Best Practices category. Within this section are the audits related to trust and safety. */
+  bestPracticesUXGroupTitle: 'User Experience',
+  /** Title of the Browser Compatibility group of the Best Practices category. Within this section are the audits related to trust and safety. */
```

Comment:
```suggestion
  /** Title of the Browser Compatibility group of the Best Practices category. Within this section are the audits related to whether the page is interpreted consistently by browsers. */
```

Reply:
For the record, I support calling this `Browser Compatibility` because all we do right now is basically ensure the page is interpreted consistently by chrome with very minor cross-browser benefits. If this morphs into more cross browser advice like we hope then we can retitle the label 👍